server/src/test/java/com/sauron/service/TransactionServiceImplTest.java

```diff
@@ -42,6 +52,19 @@ public void validIdShouldReturnDto() {
         ).contains(ACCOUNT_NUMBER, PAY, BigDecimal.TEN);
     }
 
+    @Test
+    public void shouldReturnCorrectDto() {
+        given(bankServiceExecutor.execMethod(any())).willReturn(Collections.singletonList(List.of(TRANSACTION_DTO)));
```

Comment:
I think `singletonList(TRANSACTION_DTO) ` would be enough here. You don't need `of()` method and static import would make code a bit more concise.

Reply:
This has to be here unfortunatelly (without it there will be an exception)